### ISSUE TITLE
[Request] Facebook Native Ads

### ISSUE THREAD

**Issue by technohowl** [clockbyte/admobadapter#61]
Adding this as a request to ad support for Audience Network native ads. 

@clockbyte Please take out some time to implement audience network native ads in the adapter.

**Comment by kot331107** [comment]
@technohowl We'll take a look. Bookmarking it https://developers.facebook.com/docs/audience-network/android

**Comment by hoanghiephui** [comment]
@clockbyte 
I think, this is a great request

**Comment by kot331107** [comment]
@hoanghiephui Absolutely. But a bit later I guess. You guys could contribute it if you wish, will be very appreciated I guess. And I think we can take **AdmobAdapterRecyclerWrapper** and underlying classes as a base for Facebook ads.
